File: app/src/main/java/com/example/zombiechat/src/chat/data/models/SingleChatModel.java
package com.example.zombiechat.src.chat.data.models;

import com.google.firebase.Timestamp;

public class SingleChatModel {
    String message;
    String sentTo;
    String sendBy;
    Timestamp time;

    public SingleChatModel() {
    }


    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getSentTo() {
        return sentTo;
    }

    public void setSentTo(String sentTo) {
        this.sentTo = sentTo;
    }

    public String getSendBy() {
        return sendBy;
    }

    public void setSendBy(String sendBy) {
        this.sendBy = sendBy;
    }

    public Timestamp getTime() {
        return time;
    }

    public void setTime(Timestamp time) {
        this.time = time;
    }
}

File: app/src/main/java/com/example/zombiechat/src/chat/data/models/SingleChatModel.kt
package com.example.zombiechat.src.chat.data.models

import com.google.firebase.Timestamp

class SingleChatModel {
    @JvmField var message: String? = null
    @JvmField var sentTo: String? = null
    @JvmField var sendBy: String? = null
    @JvmField var time: Timestamp? = null
}
